sentry-core/src/profiling.rs

```diff
@@ -221,6 +228,8 @@ fn get_profile_from_report(
             });
     }
 
+    samples.retain(|s| thread_samples.get(&s.thread_id).unwrap() > &1);
```

Comment:
```suggestion
    samples.retain(|s| thread_samples.get(&s.thread_id).unwrap_or_default() > &1);
```

Reply:
@Swatinem does it actually work in this case?

The compiler suggests it's not for `&u64`:
```
the trait bound `&u64: std::default::Default` is not satisfied
the trait `std::default::Default` is implemented for `u64`
```

Reply:
Ah, you can do a `.get().copied().unwrap_or_default() > 1` in that case.

I mean the `unwrap` itself is pretty much unreachable, though the compiler can’t know that :-)